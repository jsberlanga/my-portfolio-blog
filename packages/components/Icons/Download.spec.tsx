import * as React from 'react';
import { render } from '@testing-library/react';
import { Download } from './Download';

describe('Download', () => {
  it('renders correctly', () => {
    const { container } = render(<Download />);

    expect(container).toMatchSnapshot();
    expect(container).toHaveTextContent('download');
  });
});
