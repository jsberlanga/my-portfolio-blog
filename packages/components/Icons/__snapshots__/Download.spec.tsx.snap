// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Download renders correctly 1`] = `
<div>
  <svg
    data-name="Layer 1"
    fill="var(--c-light)"
    viewBox="0 0 100 100"
    width="2.2rem"
    x="0px"
    xmlns="http://www.w3.org/2000/svg"
    y="0px"
  >
    <title>
      download resume
    </title>
    <path
      d="M72.81,22A28,28,0,0,0,18,30v.4A20,20,0,0,0,22,70H34a4,4,0,0,0,0-8H22a12,12,0,0,1-.19-24l.49,0a4.3,4.3,0,0,0,3.1-1.4,4,4,0,0,0,.91-3.27,20,20,0,1,1,39.45-6.3,4,4,0,0,0,4.81,3.32A16.19,16.19,0,0,1,74,30a16,16,0,0,1,0,32H66a4,4,0,0,0,0,8h8a24,24,0,1,0-1.19-48Z"
    />
    <path
      d="M50,46a4,4,0,0,0-4,4V84.34l-5.17-5.17a4,4,0,0,0-5.66,5.66l12,12a4,4,0,0,0,5.66,0l12-12a4,4,0,0,0-5.66-5.66L54,84.34V50A4,4,0,0,0,50,46Z"
    />
  </svg>
</div>
`;
